packages/react-vapor/src/components/radio/examples/RadioExamples.1.use.md

```diff
@@ -0,0 +1,32 @@
+## Guidelines
+
+(See also: **Guidelines for All Form Controls** - _coming soon!_)
+
+### General
+
+-   A set should always have a title.
+-   Use a wording that indicates that the user can select **only one option**. Some user may not know the difference between radio buttons and checkboxes.
+-   Try to put the most common options at the top of the list, unless your options have a strong logical sorting order (such as prices or sizes).
+-   Keep the set under 7 options to choose from.
+
+### Label
+
+-   Try to limit the size of the label to one short line.
+-   Add a description or help hint only if necessary to explain the option.
+
+### Behavior
+
+-   Add a default preselected value using the recommended answer or, if there is none, the one most relevant to users.
+-   Never use radio buttons if they are no answer possible in the set. The user **must** provide an answer to move on.
+
+---
+
+## Related Components
+
+### [Dropdown](#/components/DropdownSearch)
+
+Use Dropdowns when you have more than 7 options to show.
+
+### [Toggle](#/components/FlatSelect)
```

Comment:
I would have suggested the _checkbox_ since we rarely use toggles.

Reply:
These are all components related to your action goal : Choice one single answer.
We'll see if we need to add more later on. But I think we should aim to keep Components with very similar use or goal referenced here. Otherwise, there will be no end to it.

Reply:
I've seen more confusion between _radio & checkbox_ than _radio & toggle_, but I get your point.